### ISSUE TITLE
gene change biotype issue

### ISSUE THREAD

**Issue by epaule** [WormBase/names#316]
i tried to change the biotype of WBGene00229914 to CDS and i got "A problem occured" back with the gene looking suppressed now. 
Is that intended?

**Comment by Paul-Davis** [comment]
This is a suppressed gene in geneace @epaule The UI isn't displaying the history correctly but it currently looks like a gene with a CDS biotype so your oporation worked so not a bug.

https://test-names.wormbase.org/gene/id/WBGene00229914

Shows the state of the gene prior to your biotype update. I will ressurect the gene.....but to double check it is no longer Transposon related then @epaule

**Comment by epaule** [comment]
thanks. Yes, please resurrect it as "coding". If you want, you can add Nic Wheeler (WBPerson46700) as reason/evidence.

**Comment by mluypaert** [comment]
**TL;DR:** error cause by historic events, nothing to fix.

I can confirm based on the Datomic history for this gene that it was already suppressed before your biotype update (it was imported as such) and that the biotype update you submitted occurred correctly in Datomic. The error received when viewing the gene history on the website (both test and prod) is probably caused by some events in this gene's history not having all required attributes in their provenance (full history is returned correctly through the API).
